crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -34,6 +34,15 @@ def check
       }
     end
 
+    def repocheck
+      response = {}
+      addons = Api::Crowbar.new.addons
+      addons.push("os").push("openstack").each do |addon|
```

Comment:
What about `addons.concat([ "os", "openstack"]).each` ?

(But I do not really care)